test/integration/controller_test.go

```diff
@@ -86,14 +91,102 @@ func TestAddAndRemoveBroker(t *testing.T) {
 			Status: v1alpha1.ConditionTrue,
 		})
 	if err != nil {
-		t.Fatal(err)
+		t.Fatalf("error waiting for broker to become ready: %v", err)
 	}
 
 	err = util.WaitForServiceClassToExist(client, testServiceClassName)
 	if nil != err {
 		t.Fatalf("error waiting from ServiceClass to exist: %v", err)
 	}
 
+	// TODO: find some way to compose scenarios; extract method here for real
+	// logic for this test.
+
+	//-----------------
+
+	const (
+		testNamespace    = "test-ns"
+		testInstanceName = "test-instance"
+	)
+
+	instance := &v1alpha1.Instance{
+		ObjectMeta: v1.ObjectMeta{Namespace: testNamespace, Name: testInstanceName},
+		Spec: v1alpha1.InstanceSpec{
+			ServiceClassName: testServiceClassName,
+			PlanName:         testPlanName,
+		},
+	}
+
+	_, err = client.Instances(testNamespace).Create(instance)
```

Comment:
agree here, not below (if you refer to L125), I think that will make it unreadable :)